docs/guides/Styling the UI.md

```diff
@@ -0,0 +1,74 @@
+# Styling the UI
+
+If you're using [`NavigationMapViewController`](https://www.mapbox.com/mapbox-navigation-ios/navigation/0.18.1/Classes/NavigationViewController.html), it's possible to apply custom fonts and colors to various parts of the UI.
+
+### Create a new Style class
+
+Simply subclass `DayStyle`, and make the necessary changes.
```

Comment:
Nit: Avoid superlatives like “simply” and “even” in documentation; they run the risk of sounding condescending.